src/server/raftkv.rs

```diff
@@ -325,34 +326,43 @@ where
     type Snap = RegionSnapshot<E::Snapshot>;
     type Local = E;
 
-    fn kv_engine(&self) -> E {
-        self.engine.clone()
+    fn kv_engine(&self) -> Option<E> {
+        Some(self.engine.clone())
```

Comment:
Is there a followup PR? Because raftkv should be multi-rocksdb.

Reply:
There will likely be a new v2 RaftKv struct which imlements `Engine` where no this kv_engine shoud be existed and it returns `None` instead.

Reply:
Is it possible to construct the `Engine` using tablet factory? Using something similar to `TestTabletFactory` can adapt to singleton case.

Reply:
Use tablet factory is one choice, but after the PR https://github.com/tikv/tikv/pull/13322, we may no longer need tablet factory.

Reply:
Main thing is why introducing a RaftKvV2, isn't it used to hold a tablet factory?

Reply:
I don't know exactly why introducing RaftKvV2. @BusyJay

Reply:
No, v2 will not have a tablet factory.